Code/GraphMol/SubstructLibrary/Wrap/rough_test.py

```diff
@@ -616,6 +616,50 @@ def testPropHolder(self):
       indices = library.GetMatches(m)
       self.assertEqual(['Z11234'], list(library.GetKeyHolder().GetKeys(indices)))
 
-
+  def test_bad_smiles(self):
```

Comment:
This is really horribly verbose when it runs and fills the console in a way that makes it hard to figure out what's going on.
Any chance we can reduce the apparent verbosity of this by using the Python logger and then disabling that log? That would still make all the logging calls desired but wouldn't pollute the logs quite so badly.